examples/basic/cohere.yaml

```diff
@@ -0,0 +1,87 @@
+# Copyright Envoy AI Gateway Authors
```

Comment:
can you add a line to https://github.com/envoyproxy/ai-gateway/blob/main/examples/basic/README.md to mention this file?